### PR TITLE
attribute autocapitalize

### PR DESCRIPTION
**Pull request by darlanmendonca**
implementation and some specs to attribute autocapitalize, following specs from https://github.com/minimalist-components/webcomponents/issues/17

### CODE REVIEW

**Comment by darlanmendonca** [comment]
@alencar13 review pls, testing behavior in mobile (manually)

**Comment by alencar13** [comment]
I added more one spec. Please take a look.

File: webcomponents/input/input.webcomponent.spec.js [hunk 1]
```diff
@@ -224,8 +224,8 @@ describe('mn-input (webcomponent)', () => {
     it('should remove attribute from child input', () => {
```
Comment (darlanmendonca) [inline webcomponents/input/input.webcomponent.spec.js:224]: nice!!

just change the description pls

File: webcomponents/input/input.webcomponent.spec.js [hunk 2]
```diff
@@ -249,6 +249,11 @@ describe('mn-input (webcomponent)', () => {
       element.setAttribute('autocapitalize', 'characters')
       expect(element.querySelector('input')).to.have.attribute('autocapitalize', 'characters')
     })
+    it('should be the last status to autocapitalize', () => {
```
Comment (darlanmendonca) [inline webcomponents/input/input.webcomponent.spec.js]: maybe "should change the attribute"

Comment (alencar13) [reply webcomponents/input/input.webcomponent.spec.js]: can be, i'll change.

File: webcomponents/input/input.webcomponent.spec.js (hunk 1 above)
Comment (alencar13) [reply webcomponents/input/input.webcomponent.spec.js:224]: Changed

**Review by alencar13 — APPROVED** [review]

**Pull request merged**